lib/AtomsIOPython/src/ase.jl

```diff
@@ -11,26 +11,48 @@ Supported formats:
   - ASE trajectory files
   - [XYZ](https://openbabel.org/wiki/XYZ) and [extxyz](https://github.com/libAtoms/extxyz#extended-xyz-specification-and-parsing-tools) files
 """
-struct AseParser <: AbstractParser end
+@kwdef struct AseParser <: AbstractParser 
+    read::Bool = true
+    guess::Bool = true
+end
 
 
-function AtomsIO.supports_parsing(::AseParser, file; save, trajectory)
+function AtomsIO.supports_parsing(parser::AseParser, file; save, trajectory)
     format = ""
     try
-        format = ase.io.formats.filetype(file; read=!save, guess=false)
+        format = ase.io.formats.filetype(file; read=parser.read, guess=parser.guess)
     catch e
         e isa PyException && return false
         rethrow()
     end
 
     if !(format in ase.io.formats.ioformats)
         return false
-    elseif trajectory
-        # Loading/saving multiple systems is supported only if + in code
-        supports_trajectory = '+' in ase.io.formats.ioformats[format].code
-        return supports_trajectory
-    else
-        return true
+    end
+
+    ioformat = ase.io.formats.ioformats[format]
+    supports_trajectory = '+' in ase.io.formats.ioformats[format].code
+
+    if save # Check whether ASE can write format
+        if Bool(ioformat.can_write)
```

Comment:
pyconvert(Bool, ioformat.can_write)

Reply:
That would require making PythonCall a dependency, which is why I did this. Is that fine? Currently ASEconvert is used to import ASE.

Reply:
I think ASEconvert export pyconvert.

Reply:
Yes it does:
https://github.com/mfherbst/ASEconvert.jl/blob/master/src/ASEconvert.jl#L10

Reply:
Oh I didn't check that. Thanks. Will change it.